CMakeLists.txt

```diff
@@ -198,6 +198,22 @@ add_subdirectory(cluster)
 add_subdirectory(library)
 add_subdirectory(vowpalwabbit)
 
+# Add natvis file if the Generator is Visual Studio
+if (CMAKE_VERSION VERSION_GREATER 3.7.8)
+    if (MSVC_IDE)
+        option(VS_ADD_NATIVE_VISUALIZERS "Configure project to use Visual Studio native visualizers" TRUE)
```

Comment:
What is the reasoning behind making this an option? Is there a time when you would not want to do this, or to force it to be on?